lib/eventFactory.js

```diff
@@ -205,7 +205,7 @@ function createBuilds(config) {
 
     return Promise.all([
         pipeline.branch,
-        pipeline.jobs
+        pipeline.getJobs()
```

Comment:
should we pass in `type` based on `isPR`?

Reply:
yeah, we can but just found out that the getJobs is fetching all the jobs anyway. And if you set the type, it will parse the list and filter it out. So don't think the extra logic will help a lot...

Reply:
ah okay